apps/reference/docs/guides/platform/backups.mdx

```diff
@@ -0,0 +1,102 @@
+---
+id: backups
+title: Backups
+description: Available forms of backups for your Supabase project
+---
+
+## Database Backups
```

Comment:
style nit: I don't think we need a heading immediately following the title
![Screen Shot 2022-10-26 at 3 01 16 PM](https://user-images.githubusercontent.com/7026076/198146946-46af573b-7d68-4597-8f4c-44ed3582c316.png)

```suggestion
---
id: database-backups
title: Database Backups
description: Learn about the different ways your Supabase project's database is backed up
---
```